### PR TITLE
Fizzbuzz feature(Caesar Cipher) done

### PR DESCRIPTION
**Pull request by alexholft**

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+print('it works!')
```
Comment (hwanlloween) [inline fizzbuzz.py]: wow amazing!!

File: fizzbuzz.py [hunk 2]
```diff
@@ -1,2 +1,21 @@
-print('it works!')
```
Comment (hwanlloween) [inline fizzbuzz.py]: looks so nice

**Review by hwanlloween — APPROVED** [review]

File: fizzbuzz.py (hunk 1 above)
Comment (alexholft) [reply fizzbuzz.py]: thank you!

File: fizzbuzz.py (hunk 2 above)
Comment (alexholft) [reply fizzbuzz.py]: thank you.

File: fizzbuzz.py [hunk 3]
```diff
@@ -1,2 +1,21 @@
...
+      else:
+         result += chr((ord(char) + s - 97) % 26 + 97)
+      return result
+#check the above function
+text = "CEASER CIPHER DEMO"
+s = 4
+
+print "Plain Text : " + text
+print "Shift pattern : " + str(s)
+print "Cipher: " + encrypt(text,s)
```
Comment (hwanlloween) [inline fizzbuzz.py:20]: caesar cipher wow..it's so hard..

**Review by hwanlloween — CHANGES_REQUESTED** [review]

**Pull request merged**